tests/unit/config/test_configcommands.py

```diff
@@ -511,14 +512,15 @@ def test_config_source(self, tmpdir, commands, config_stub, config_tmpdir,
 
     def test_config_py_arg_source(self, commands, config_py_arg, config_stub):
         assert config_stub.val.content.javascript.enabled
-        config_py_arg.write_text('c.content.javascript.enabled = False\n',
+        config_py_arg.write_text('\n'.join(['config.load_autoconfig(False)',
+                                            'c.content.javascript.enabled = False']),
                                  encoding='utf-8')
         commands.config_source()
         assert not config_stub.val.content.javascript.enabled
 
     def test_errors(self, commands, config_tmpdir):
         pyfile = config_tmpdir / 'config.py'
-        pyfile.write_text('c.foo = 42', encoding='utf-8')
+        pyfile.write_text('config.load_autoconfig(False)\nc.foo = 42', encoding='utf-8')
```

Comment:
I'd prefer using `'\n'.join` here as well to make it a bit more readable (and consistent).